api/src/main/resources/workbench.yaml

```diff
@@ -294,6 +294,28 @@ paths:
           schema:
             $ref: "#/definitions/ErrorResponse"
 
+  /v1/update-nih-token:
+    post:
+      tags:
+        - profile
+      consumes:
+        - application/json
+      description: Updates a users NIH token
+      operationId: updateNihToken
+      parameters:
+        - in: body
+          name: token
+          description: the token retrieved from NIH
+          schema:
+            $ref: "#/definitions/NihToken"
+      responses:
```

Comment:
Should this return the updated profile? Or do we not expect strong consistency on the profile being updated?

Reply:
There's no reason **not** to return the full profile, but I was not envisioning that being a necessity either, i.e. it isn't necessary for the usage in this PR. Would be happy to make that a return if we think it will be helpful in the future.

Reply:
Sounds fine.

Reply:
Added